src/notation/view/widgets/measureproperties.cpp

```diff
@@ -77,8 +77,13 @@ MeasurePropertiesDialog::MeasurePropertiesDialog(QWidget* parent)
     setFocus();
 
     qApp->installEventFilter(this);
+
+    if (!m_measure) {
+        deleteLater();
```

Comment:
tried various methods to ensure dialog wouldn't show at all if no valid measure selected, this was the only one that didn't crash (though it does cause the dialog frame to flicker briefly - note that the plan is to replace this dialog with the properties panel anyway)

Reply:
I propose to solve this problem in another way:
The NotationActionController determines whether a measure is selected and if it is not selected, then do not show the dialog, but show a warning that a measure is not selected. 

Also, there is assert in dev mode if m_measure isn't selected
`    IF_ASSERT_FAILED(measure) {
        return;
    }
    `

Reply:
I believe I've addressed this concerned now? The crash definitely still happens (I just lost work because of it).